gazelle/BUILD.bazel

```diff
@@ -20,3 +21,47 @@ gazelle(
     ],
     command = "update-repos",
 )
+
+go_library(
+    name = "gazelle_library",
+    srcs = glob(
+        ["*.go"],
```

Comment:
Why use a glob by hand here? Just use the Gazelle extension for Go. Check how we have it setup in the rules_python repository.

Reply:
Done 👍